component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/util/BackupCodeUtil.java

```diff
@@ -443,5 +446,23 @@ public static String getBackupCodeAuthenticatorConfig(String key, String tenantD
                     ERROR_CODE_ERROR_GETTING_CONFIG.getMessage(), e);
         }
     }
+
+    /**
+     * Generate the hash value of the given string.
+     *
+     * @param backupCode String value that needs to hash.
+     * @return Hash value of the backupCode.
+     * @throws NoSuchAlgorithmException If the algorithms is invalid.
```

Comment:
Let's not throw the NoSuchAlgorithmException from here.

Reply:
addressed with [7d9674b](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/7d9674b6bf172b43c30479b0d64fdfa6f8130373)